v3/src/components/common/edit-formula-modal.tsx

```diff
@@ -86,20 +92,44 @@ export const EditFormulaModal = observer(function EditFormulaModal({
     event.stopPropagation()
   }
 
+  const handleResizeModal = useCallback((e: React.PointerEvent) => {
+    if (e.pointerId !== undefined) {
+      e.currentTarget.setPointerCapture(e.pointerId)
+    }
+
+    const modalRect = document.querySelector(".codap-modal-content")?.getBoundingClientRect()
+    const startWidth = modalRect?.width ?? dimension.width
+    const startHeight = modalRect?.height ?? dimension.height
+    const startPosition = {x: e.pageX, y: e.pageY}
+
+    let resizingWidth = startWidth, resizingHeight = startHeight
+
+    const onPointerMove = (pointerMoveEvent: { pageX: number; pageY: number }) => {
+      const xDelta = pointerMoveEvent.pageX - startPosition.x
+      const yDelta = pointerMoveEvent.pageY - startPosition.y
+      resizingWidth = Math.max(startWidth + xDelta, minWidth)
+      resizingHeight = Math.max(startHeight + yDelta, minHeight)
+      setDimension({
+        width: resizingWidth, height: resizingHeight,
+      })
```

Comment:
I recommend rounding these values to integer pixels -- they're much easier to deal with when debugging and I don't think fractional pixel widths are necessary for users.